### DISCUSSION TITLE
Add css

### DISCUSSION THREAD

**Discussion by SMsajjadSM** [11ty/eleventy#3117] [Q&A]
Hi first i want say tnx to Eleventy team
after that
i want Add css to my blog and i cant plz tell me step by step and speak clearly bcs it possible i cant undrstnd 
tnx

**Comment by Snapstromegon** [comment]
There are many options to achieve this, but the easiest two are either to just write your css inline e.g. in your layout. This could like this (simplified):
```njk
<!-- layout.njk -->

<html>
  <head>
    <style>
/* your css here */
    </style>
  </head>
  <body>
    {{content}}
  </body>
</html>
```

Or the second option is to add your css to a static folder and add a passthrough copy to it in your eleventy config: https://www.11ty.dev/docs/copy/

The options are also described in the docs here: https://www.11ty.dev/docs/assets/
